src/api.js

```diff
@@ -112,6 +112,38 @@ function resolve(res, status, message) {
   return res.status(status).send(message);
 }
 
+/***
+ Helper function to look up repo owner in the Azure table to get installation ID,
+ and authenticate with GitHub using that ID.
+
+ Receives owner's name, the Azure table, and github object.
+ Returns either authenticated github object or null
+***/
+async function installationAuthenticate(owner, directory, github) {
+  // Look up the installation ID in Azure. If no such owner in the table, no error thrown
+  let ownerInfo = await directory.load({owner}, true);
+  if (ownerInfo) {
+    let instGithub = await github.getInstallationGithub(ownerInfo.installationId);
+    return instGithub;
+  } else {
+    return null;
+  }
+}
+
+/***
+ Helper function to find the most fresh status set by our bot.
+ Gets the bot's ID, gets statuses for the repo/branch, finds there the status by the bot's ID
+ 
+ Receives authenticated github object; names of owner, repo and branch; and configuration object
+ Returns either status object or undefined (if not found).
+***/
+async function findTCstatus(github, owner, repo, branch, configuration) {
```

Comment:
This is super-minor, but the first `s` in `status` should be capitalized (`Status`)